internal/uvm/create.go

```diff
@@ -70,6 +70,12 @@ type Options struct {
 	// ExternalGuestConnection sets whether the guest RPC connection is performed
 	// internally by the OS platform or externally by this package.
 	ExternalGuestConnection bool
+
+	//DisableCompartmentNamespace sets whether to disable namespacing the network
+	// compartment in the UVM for WCOW. This is to work around the fact that the
+	// change is gated behind a registry key currently. Once this is removed this
+	// can also be removed.
```

Comment:
"the change" is ambiguous, can you elaborate more? 

Then the last sentence would be more clear (at least to me) if it said something more like "Once the change is no longer gated by a registry key, we can remove the need for this variable and the associated annotation."

Reply:
done

Reply:
Well actually besides the last sentence. Will fix also